mifosng-android/src/main/java/com/mifos/mifosxdroid/online/runreports/reportcategory/ReportCategoryFragment.java

```diff
@@ -60,6 +74,7 @@ public static ClientReportCategoryFragment newInstance() {
     public void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         ((MifosBaseActivity) getActivity()).getActivityComponent().inject(this);
+        getActivity().registerReceiver(this.broadCastNewMessage, new IntentFilter("report"));
```

Comment:
It'll be better to keep this `action` too in the Constants file.

Reply:
@tarun0  done